test/integration/features/basic.feature

```diff
@@ -27,6 +31,8 @@ Feature: Basic
       hello
       """
 
+  Scenario: Minishift --docker-env works properly
```

Comment:
BTW, this is a bit unfortunate, since this scenario depends on the fact how Minshift was started earlier. If it were not so expensive to start/stop Minishift, this really should go into its own feature/suite. 

Moving forward, I could see that we have a docker.feature, were we test various other things around Docker, eg pushing to the exposed OpenShift registry. In this case I think it makes sense to move this scenario into the Docker feature.